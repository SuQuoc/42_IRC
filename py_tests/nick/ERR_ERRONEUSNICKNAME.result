:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 432 client0 123456789abc :Erroneus nickname
:AfterLife 431 client0 :No nickname given
:AfterLife 431 client0 :No nickname given
:AfterLife 432 client0 clie,nt0 :Erroneus nickname
:AfterLife 432 client0 clie@nt0 :Erroneus nickname
:AfterLife 432 client0 clie!nt0 :Erroneus nickname
:client0!user0@10.14.3.7 NICK :space
:AfterLife 433 space space :Nickname is already in use
:space!user0@10.14.3.7 NICK :space0
:space0!user0@10.14.3.7 NICK :Nspace
