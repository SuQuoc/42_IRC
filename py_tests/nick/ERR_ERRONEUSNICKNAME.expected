:AfterLife 001 client0 :Welcome to the Internet Relay Network, user0
:AfterLife 005 client0 NICKLEN=9 CHANTYPES=#& CHANMODES=oiklt :are supported by this server
:AfterLife 432 client0 123456789abc :Erroneus nickname
:AfterLife 431 client0 :No nickname given
:AfterLife 431 client0 :No nickname given
:AfterLife 432 client0 clie,nt0 :Erroneus nickname
:AfterLife 432 client0 clie@nt0 :Erroneus nickname
:AfterLife 432 client0 clie!nt0 :Erroneus nickname
:client0!user0@10.12.3.9 NICK :space
:AfterLife 433 space space :Nickname is already in use
:space!user0@10.12.3.9 NICK :space0
:space0!user0@10.12.3.9 NICK :Nspace
:AfterLife 431 Nspace :No nickname given
